pom.xml

```diff
@@ -28,7 +28,7 @@
     <repository>
       <id>xmldoclet</id>
       <name>XML Doclet</name>
-      <url>https://github.com/com.github.markusbernhardt/xml-doclet/raw/repository/</url>
+      <url>https://raw.githubusercontent.com/szirbucz/xml-doclet-1/repository/</url>
```

Comment:
Since I don't have write acces to the edemo/PDEngine repo, I uploaded the needed maven artifacts to my own.
A better solution would be the following:
- create a repository branch for PDEngine
- install all the artifacts we need to the repository branch
- use https://raw.githubusercontent.com/edemo/PDEngine/repository/ here